spec/repository_spec.rb

```diff
@@ -129,21 +197,47 @@
         expect(chef_run).to install_package('gnupg') if chef_run.node['packages']['gnupg2'].nil?
       end
 
-      it 'downloads the new RPM key' do
+      it 'downloads the new RPM key E09422B3' do
         expect(chef_run).to create_remote_file('DATADOG_RPM_KEY_E09422B3.public').with(path: ::File.join(Chef::Config[:file_cache_path], 'DATADOG_RPM_KEY_E09422B3.public'))
       end
 
-      it 'notifies the GPG key install if a new one is downloaded' do
+      it 'notifies the GPG key install if a new one is downloaded E09422B3' do
         keyfile_r = chef_run.remote_file('DATADOG_RPM_KEY_E09422B3.public')
         expect(keyfile_r).to notify('execute[rpm-import datadog key e09422b3]')
           .to(:run).immediately
       end
 
-      it 'doesn\'t execute[rpm-import datadog key e09422b3] by default' do
+      it 'doesn\'t execute[rpm-import datadog key *] by default E09422B3' do
         keyfile_exec_r = chef_run.execute('rpm-import datadog key e09422b3')
         expect(keyfile_exec_r).to do_nothing
       end
 
+      # Key 20200908
```

Comment:
```suggestion
      # Key FD4BF915 (2020-09-08)
```

to make this clearer.